Source/EasyNetQ/Internals/TaskHelpers.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace EasyNetQ.Internals
 {
-    public static class TaskHelpers
+    internal static class TaskHelpers
```

Comment:
This class is about magic and it will be better to not use it anywhere else...
And also in netstandard(and from 4.6) configuration there is no reason to use this class, because in standard library we already have (Task.FromException)[https://msdn.microsoft.com/ru-ru/library/dn823324(v=vs.110).aspx] and TaskHelpers.ExecuteSynchronously can be replaced with 
```
private async Task Task.ExecuteSynchronously(Action action)
{
    action.Invoke();
    await Task.CompletedTask;
}
```